LocalStorage.coffee

```diff
@@ -92,6 +101,9 @@ class LocalStorage
       @metaKeyMap[key] = metaKey
       @length += 1
       @bytesInUse += valueStringLength
+    if hasListeners
+      evnt = new StorageEvent key, oldValue, value
```

Comment:
Should this have `filename` added to the last @url parameter?

Also, I know it's not canonical CoffeeScript, but could you please add parens to this (and other function calls). That's my personal style guide for CoffeeScript.